package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jaas",
+  "version": "0.1.0",
+  "description": "Jokes as a Service",
+  "main": "index.js",
+  "scripts": {
+    "test": "./bin/test.sh",
+    "start": "node ./app/server.js | bistre",
+    "lint": "standard"
+  },
+  
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stephenrlouie/JaaS.git"
+  },
+  "author": "Stephen Louie <stelouie@cisco.com>",
+  "contributors": [
+    "Michael Napolitano <mnapolit@cisco.com>",
+    "Adam Schaub <adamscha@cisco.com>"
+  ],
+  "dependencies": {
+    "bole": "2.0.0",
+    "express": "4.13.4",
+    "jade": "1.11.0",
```

Comment:
I'm seeing jade has been deprecated.

```
npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade
```